etc/databases/world/updates/updates.sql

```diff
@@ -3291,6 +3291,34 @@ begin not atomic
         insert into`applied_updates`values ('310720231');
     end if;
 
+    --01/08/2023
```

Comment:
![image](https://github.com/The-Alpha-Project/alpha-core/assets/8654362/5cfdbdc2-508d-4bf6-8d23-3e71c4d1901f)
Please use the same format.